packages/toolkit/src/query/react/ApiProvider.tsx

```diff
@@ -37,6 +38,13 @@ export function ApiProvider<A extends Api<any, {}, any, any>>(props: {
   setupListeners?: Parameters<typeof setupListeners>[1] | false
   context?: Context<ReactReduxContextValue>
 }) {
+  const context = props.context || ReactReduxContext
+  const existingContext = useContext(context)
+  if (existingContext) {
+    throw new Error(
+      'Existing Redux context detected. If you already have a store set up, please use the traditional Redux setup.'
```

Comment:
:hand: This should result in a new error code being added.  Can you run `yarn build` locally, see if it updated `errors.json`, and push that if there is a change?

Reply:
yep, done